port/unix/omrsysinfo.c

```diff
@@ -1189,8 +1189,31 @@ omrsysinfo_get_number_CPUs_by_type(struct OMRPortLibrary *portLibrary, uintptr_t
 			}
 		}
 
+		/* If system calls failed to retrieve info, do not check cgroup at all, and give an error */
 		if (0 == toReturn) {
 			Trc_PRT_sysinfo_get_number_CPUs_by_type_failedBound("errno: ", errno);
+		} else if (portLibrary->sysinfo_cgroup_are_subsystems_enabled(portLibrary, OMR_CGROUP_SUBSYSTEM_CPU)) {
+			int32_t rc = 0;
+			int64_t cpuQuota = -1;
+			uint64_t cpuPeriod = 100000;
```

Comment:
These are the default values of `cpu.cfs_quota_us` and `cpu.cfs_period_us`. They indicate how much time (quota, in microseconds) out of a given period (also in microseconds) can a CPU spend on the process (the -1 means no limit). [See here for a complete summary of these values](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/sec-cpu). 

 Initially this code would continue if it failed to read either of the values, so setting defaults was necessary, but since one is meaningless without the other, it was changed to break out on failure.

To avoid confusion, I can change the variables to not be initialized or initialize to 0 and add a comment briefly explaining what they mean. Or should I just indicate they are defaults?

Reply:
I would likely just initialized them to a default value of 0 or something similar as to remove the confusion that these values may be of significance.